optimize/dataio.py

```diff
@@ -30,21 +30,33 @@ def __init__(self, filename, swap_xz=True):
             tracks['z_end'] = x_end
             tracks['z'] = x
 
+        tracks = tracks[200:250]
```

Comment:
This should probably be removed

Reply:
see commit 59b987ea1671cb42e86c9f929c059f27ffdf8ccf